x-pack/plugins/security_solution/server/lib/timeline/saved_object/timelines/field_migrator.ts

```diff
@@ -7,11 +7,13 @@
 
```

Comment:
Dear reviewer who has made it this far, you might want to pop that console open and run:
`document.querySelectorAll('.load-diff-button').forEach(node => node.click())` 
🙏